packages/pigment-css-unplugin/src/index.ts

```diff
@@ -356,10 +356,7 @@ export const plugin = createUnplugin<PigmentOptions, true>((options) => {
           code: result.code,
           map: result.sourceMap,
         };
-      } catch (e) {
-        const error = new Error((e as Error).message);
-        error.stack = (e as Error).stack;
-        throw error;
+      } catch(e) {
```

Comment:
Like stated in the pr comment that I am not sure how to show this. But regarding you will see it in the Error view in your browser.

Like my example repo

![image](https://github.com/user-attachments/assets/1fd0989b-7945-42f6-bdc3-926152b4d9fd)

![image](https://github.com/user-attachments/assets/0546386f-64b8-476d-8010-642857416262)

It is handled. So i am not really sure what this throw aims to do since it is not handle anywhere. So today it will just crash nextjs and you have to restart the process.